packages/snaps-simulator/src/features/renderer/Renderer.tsx

```diff
@@ -22,6 +22,11 @@ export const components: Partial<
   [NodeType.Spinner]: Spinner,
   [NodeType.Text]: Text,
   [NodeType.Image]: Image,
+  // @todo: Create a button
+  // TODO(@guillaumerx): Quick fix to build, update those later
```

Comment:
What is the plan for this?

Reply:
https://github.com/MetaMask/snaps/issues/2119